cli/cfncluster/cfncluster.py

```diff
@@ -251,7 +252,7 @@ def get_master_server_id(stack_name, config):
 
     return [r.physical_resource_id for r in resources if r.logical_resource_id == 'MasterServer'][0]
 
-def poll_master_server_state(stack_name, config):
+def poll_master_server_state(stack_name, config, nowait):
```

Comment:
i don't love what you've done to this function.  I think you need to refactor.  You're depending on nowait meaning "don't print", which is weird.  I'm all for reusing code, but maybe splitting it up a bit to handle both use cases would be better.